data/sql_setup/prepare_schedule_c.sql

```diff
@@ -0,0 +1,22 @@
+-- Create trigger to maintain Schedule C search columns
+ALTER TABLE fec_vsum_sched_c DROP COLUMN if EXISTS candidate_name_text;
+ALTER TABLE fec_vsum_sched_c DROP COLUMN if EXISTS loan_source_name_text;
+
+ALTER TABLE fec_vsum_sched_c add column candidate_name_text tsvector;
+ALTER TABLE fec_vsum_sched_c add column loan_source_name_text tsvector;
+
+UPDATE fec_vsum_sched_c SET candidate_name_text = to_tsvector(cand_nm);
+UPDATE fec_vsum_sched_c SET loan_source_name_text = to_tsvector(loan_src_nm);
```

Comment:
What is the purpose of having these two `update` statements here given the trigger defined below?  I'm assuming for initializing but wanted to be sure.